Bottom: 0dda77a7aec4257dc9f9bcc7914ae2be516360d5
Top:    a8302a657d92ec348757e71a68a7c2dbdb513053
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:44:31 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
